### PR TITLE
Update embeddings.py

### PR DESCRIPTION
**Pull request by soma2000-lang**
## Description
I have followed all the guidelines

## Relevant Issue
Issue #Tag the issue here

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/NimbleEdge/EnvisEdge/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [NimbleEdge Styleguide](https://numpydoc.readthedocs.io/en/latest/format.html)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/NimbleEdge/EnvisEdge/labels)
- [x] My changes are covered by tests

### CODE REVIEW

**Comment by ramesht007** [comment]
Hey @soma2000-lang ,
please follow these documentation guidelines:
pep8 [guidelines](https://www.python.org/dev/peps/pep-0008/)
numpy style [guide](https://numpydoc.readthedocs.io/en/latest/format.html)

and refactor your comments.
thanks

File: fedrec/modules/embeddings.py [hunk 1]
```diff
@@ -99,12 +111,23 @@ def __init__(
 
 @registry.load("embedding", "pr_emb")
 class PrEmbeddingBag(nn.Module):
+    '''It is like adding a extra rapper layer upon the embedding.
```
Comment (kaiserliche) [inline fedrec/modules/embeddings.py:114]: Give the exact work rather than explaining by giving analogy.

File: fedrec/modules/embeddings.py [hunk 2]
```diff
@@ -99,12 +111,23 @@ def __init__(
...
+        Inputs
+        nn.Module:Base class for all neural network modules.'''
     def __init__(self,
                  num_embeddings,
                  embedding_dim,
                  base_dim=None,
                  index=-1,
                  init=False):
+    '''Base class for all neural network modules.Used for the function definition
```
Comment (kaiserliche) [inline fedrec/modules/embeddings.py:123]: Add proper spaces after full stop.

File: fedrec/modules/embeddings.py [hunk 3]
```diff
@@ -309,12 +332,18 @@ def __init__(self, num_embeddings, embedding_dim, num_collisions,
             self.weight_r = Parameter(_weight[1])
         self.mode = mode
         self.sparse = sparse
-
+    #module for reseting the parameters
```
Comment (kaiserliche) [inline fedrec/modules/embeddings.py:335]: How is it a module?

File: fedrec/modules/embeddings.py [hunk 4]
```diff
@@ -309,12 +332,18 @@ def __init__(self, num_embeddings, embedding_dim, num_collisions,
...
     def reset_parameters(self):
         nn.init.uniform_(self.weight_q, np.sqrt(1 / self.num_categories))
         nn.init.uniform_(self.weight_r, np.sqrt(1 / self.num_categories))
 
     def forward(self, input, offsets=None, per_sample_weights=None):
+        '''The forward function computes output Tensors from input Tensors
```
Comment (kaiserliche) [inline fedrec/modules/embeddings.py:341]: It can be written in another ways which sound more professional.

File: fedrec/modules/embeddings.py [hunk 5]
```diff
@@ -309,12 +332,18 @@ def __init__(self, num_embeddings, embedding_dim, num_collisions,
...
+        Inputs
+        input:long-Takes the input
+        offsets:offsets determines the starting index position of each bag (sequence) in input.
+        per_sample_weights:
```
Comment (kaiserliche) [inline fedrec/modules/embeddings.py]: Why is it empty?

**Review by kaiserliche — CHANGES_REQUESTED** [review]

File: fedrec/modules/embeddings.py (hunk 4 above)
Comment (soma2000-lang) [reply fedrec/modules/embeddings.py:341]: @kaiserliche  Can you please guide me to write this in a more professional way.

File: fedrec/modules/embeddings.py [hunk 6]
```diff
@@ -42,11 +42,19 @@ def md_solver(n, alpha, d0=None, B=None, round_dim=True, k=None):
         undo_sort[v] = i
     return d[undo_sort]
 
-
+#defining the alpha
```
[the remark this answers is not in the document]
Comment (soma2000-lang) [reply fedrec/modules/embeddings.py:45]: @kaiserliche  I was unable to understand what this alpha was doing here. Can you provide me some context on it so that I can do it in a proper manner.

**Comment by soma2000-lang** [comment]
@kaiserliche  Thank you for reviewing. Apart  from this would you like me to some more changes ??

File: fedrec/modules/embeddings.py (hunk 1 above)
Comment (soma2000-lang) [reply fedrec/modules/embeddings.py:114]: @kaiserliche  Can you suggest to me what could I have written here.
